### PR TITLE
Add sample test

### PR DESCRIPTION
**Pull request by chuyeow**
Just showing a way to write tests. Your call on whether worth showing.

Btw, use [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports), it adds imports for you (I think you're using gofmt).

### CODE REVIEW

File: igor.go [hunk 1]
```diff
@@ -2,17 +2,33 @@ package main
...
+func init() {
+	if v, ok := os.LookupEnv("HOTELS_API_ENDPOINT"); ok {
+		hotelsAPIEndpoint = v
+	}
+}
+
 func main() {
 	http.HandleFunc("/hello", hello)
 	http.HandleFunc("/ean", ean.List)
 	http.HandleFunc("/hotels", hotels.List)
```
Comment (chuyeow) [inline igor.go]: If anybody asks if there's something better, I'd recommend chi's router: https://github.com/go-chi/chi.

File: hotels/hotels.go [hunk 2]
```diff
@@ -78,17 +78,40 @@ type Promotion struct {
 	Priority      int64  `json:"priority"`
 }
 
-func Cheapest(w http.ResponseWriter, r *http.Request) {
-	b, err := helper.Fetch("http://127.0.0.1:5091/hotels")
+type CheapestHandler struct {
+	HotelsAPIEndpoint string
+}
```
Comment (chuyeow) [inline hotels/hotels.go]: I like making my HTTP handlers full-blown structs so I can specify my dependencies (in this case, the hotels API endpoint URL is a dependency).

Then, in my tests, I can set my dependencies to mocks, but in my `func main()`, I use real values.

Comment (ctrlaltreboot) [inline hotels/hotels.go]: Ah yeah, I saw this signature back in N2O. Thanks!

**Comment by ctrlaltreboot** [comment]
👍 Who am I to judge?

**Pull request merged**